libnavigation-base/src/main/java/com/mapbox/navigation/base/extensions/NullEx.kt

```diff
@@ -0,0 +1,80 @@
+package com.mapbox.navigation.base.extensions
```

Comment:
These are not extensions. What about calling this file `NullUtils`?

Reply:
This is duplicated and available in `libnavigation-util` module https://github.com/mapbox/mapbox-navigation-android/blob/master/libnavigation-util/src/main/java/com/mapbox/navigation/utils/extensions/NullEx.kt If we're going to expose it from `libnavigation-base` should we remove the `NullEx.kt` file from `libnavigation-util`?

Reply:
I would keep this as a separate discussion. I am much more inclined to have the entire `libnavigation-utils` module inside `libnavigation-base`. It doesn't makes sense to me to have it as a separate module and then each module repeating the same class. I would suggest not to remove the `NullExt.kt`. All other modules depend on `libnavigation-base` already. Having utils inside `base` makes much more sense to me.

However, whatever the decision is, this change is not related to this PR.